wlog-zerolog/internal/logger.go

```diff
@@ -228,10 +227,6 @@ func logOutput(newEvt func() *zerolog.Event, msg, levelVal string, params []wlog
 	if !entry.evt.Enabled() {
 		return
 	}
-	entry.evt = entry.evt.Str(wlog.TimeKey, time.Now().Format(time.RFC3339Nano))
-	if levelVal != "" {
-		entry.evt = entry.evt.Str(svc1log.LevelKey, levelVal)
```

Comment:
This removal makes it such that the `levelVal` argument is no longer used -- can you remove it?